ui/imports/shared/views/chat/CompactMessageView.qml

```diff
@@ -164,8 +168,9 @@ Item {
         active: typeof root.messageContextMenu !== "undefined"
         sourceComponent: Component {
```

Comment:
Eww.... Connections inside a Loader :open_mouth: Please remove the Loader, totally unnecessary (add the Loader's `active` condition to `Connections.enabled`)